### PR TITLE
Adds Docs for Setting Up Gauntlet

### PR DESCRIPTION
**Pull request by chris-de-leon-cll**

### CODE REVIEW

File: packages-ts/starknet-gauntlet/README.md [hunk 1]
```diff
@@ -1 +1,116 @@
...
+   access_controller:deploy:multisig
+
+ocr2:
+   ocr2:deploy
+...
+```
+
+## Selecting an RPC URL
+
+Before executing a Gauntlet command, you'll need a Starknet RPC v7 URL. You can find a list of common ones [here](https://www.starknetjs.com/docs/next/guides/connect_network/). Take note of one of these URLs and save it for later - we'll need it when we move onto configuring environment variables for Gauntlet.
```
Comment (augustbleeds) [inline packages-ts/starknet-gauntlet/README.md:54]: I think op-core would be using our private RPCs for this

Comment (chris-de-leon-cll) [reply packages-ts/starknet-gauntlet/README.md:54]: Sounds good! I'll add a note on this

Comment (augustbleeds) [reply packages-ts/starknet-gauntlet/README.md:54]: you should create a new file in packages-ts/starknet-gauntlet-cli/networks called .env.mainnet.

you can initialize it with a publicly available RPC. but if they'd like to use the --mainnet flag for the gauntlet commands you can add instructions that tell them to change the .env.mainnet or .env.testnet file to a private RPC if they'd like (just a suggestion though)

Also, while you're doing that you should also change https://github.com/smartcontractkit/chainlink-starknet/blob/augustus.BCI-3299.build-report/packages-ts/starknet-gauntlet-cli/networks/.env.testnet#L1 to use v7.

File: packages-ts/starknet-gauntlet/README.md [hunk 2]
```diff
@@ -1 +1,116 @@
...
+
+## Setting up a Wallet
+
+Gauntlet commands that send transactions to the chain require a funded Starknet wallet to cover transaction fees. If you're using testnet or mainnet, you can create a Starknet wallet by following the guide in the official documentation [here](https://docs.starknet.io/documentation/quick_start/set_up_an_account/). If you're using a docker container hosting a local Starknet node (i.e. see `./scripts/devnet.sh`), then you can use one of the predeployed accounts from the container logs.
+
+## Funding your Wallet
+
+If you're using testnet, you can fund your wallet using a Starknet faucet - a popular option is [this](https://starknet-faucet.vercel.app/) one. You can also reach out on Slack to have the team send some funds directly to your account.
```
Comment (augustbleeds) [inline packages-ts/starknet-gauntlet/README.md]:
```suggestion
If you're using testnet, you can fund your wallet using a Starknet faucet - a popular option is [this](https://starknet-faucet.vercel.app/) one. You can also reach out on Slack (#team-blockchain-integrations) to have the team send some funds directly to your account.
```

**Review by augustbleeds — APPROVED** [review]

File: packages-ts/starknet-gauntlet/README.md (hunk 1 above)
Comment (augustbleeds) [reply packages-ts/starknet-gauntlet/README.md:54]: actually, you may not need to add the instructions to change .env.mainnet file if you envision op-core just using the .env file for their setup

Comment (chris-de-leon-cll) [reply packages-ts/starknet-gauntlet/README.md:54]: Yup - added some clarifications in Slack

Marking as resolved

**Pull request merged**